bn-models/src/models/_base/base-operation.ts

```diff
@@ -1,5 +1,29 @@
-// Base Operation Class, intended to be subclassed
-// Hauls, Sets, etc
+/*
+Base Operation Class, intended to be subclassed
+
+Hauls, Sets, etc., store common attributes between
+different types of operations here (currently none to store)
+*/
 import { Base } from './base';
 
-export type BaseOperation = Base
+export type BaseOperation = Base;
+/*
+  Attributes from extended interfaces are accessible in this interface.
+  
+  BaseOperation <-- Base
+
+      Base attributes:
```

Comment:
Good idea, but don't think this is necessary since people can ctrl click on base and see this info there.